ztp/ztp-ran-policy-generator/ranGenTemplateExamples/common-ranGen.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: ranPolicyGenerator/v1
+kind: RanGenTemplate
+metadata:
+  # The name will be used to generate the placementBinding and placementRule names as {name}-placementBinding and {name}-placementRule
+  name: "common-policies"
+  namespace: "policy-template"
+  labels:
+    common: true
+    groupName: "N/A"
+    siteName: "N/A"
+    mcp: "N/A"
+sourceFiles:
+  # Create operators policies that will be installed in all clusters
+  - fileName: SriovSubscription
+    name: "N/A"
```

Comment:
If we leave "name" out of this file will RanPolicyGen treat it as "N/A"? If so (or if it can) it would make this file much cleaner which simplifies our explanations to customers.

Reply:
yes, the plugin handle if the name not set. it will use the name exist in the source policy file